docs/faq.rst

```diff
@@ -406,12 +410,10 @@ InventoryItem(name='foobar', price=5)
 .. skip: end
 
 In addition, two helper functions are available to interact with items without
-having to check their base class. These functions are:
+having to check their base class:
 
 .. autofunction:: scrapy.utils.datatypes.get_item_field
 
-and
-
 .. autofunction:: scrapy.utils.datatypes.set_item_field
```

Comment:
Purely :lipstick: aesthetic, to avoid an “and” paragraph between the API documentation of each function.